pros/src/competition.rs

```diff
@@ -21,21 +23,30 @@ impl CompetitionStatus {
 }
 
 /// Get the current status of the robot.
+#[deprecate_until(remove = ">= 0.7", note = "use `status` instead")]
 pub fn get_status() -> CompetitionStatus {
+    status()
+}
+
+/// Get the current status of the robot.
+pub fn status() -> CompetitionStatus {
     CompetitionStatus(unsafe { pros_sys::misc::competition_get_status() })
 }
 
 /// Check if the robot is in autonomous mode.
+#[deprecate_until(remove = ">= 0.7", note = "use `status().autonomous()` instead")]
 pub fn is_autonomous() -> bool {
-    unsafe { pros_sys::misc::competition_is_autonomous() }
+    status().autonomous()
 }
 
 /// Check if the robot is disabled.
+#[deprecate_until(remove = ">= 0.7", note = "use `status().disabled()` instead")]
```

Comment:
These should have the parameter `remove = ">= 1.x"` to comply with semver (removing function is breaking change). The remove argument just prevents building the library if the version matches the pattern so maintainers remember to actually remove it in the future before release.

Reply:
Waiting until v1.0 to remove this is not required because the project is in the [development phase](https://semver.org/#spec-item-4):

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

As such, minor version releases can contain backward-incompatible changes.